### PR TITLE
PR of my_branch

### PR DESCRIPTION
**Pull request by joeg3**
My awesome changes

### CODE REVIEW

File: b.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Update from branch my_branch
```
Comment (jglin) [inline b.txt]: Add an exclamation point to this ine

File: b.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
 Update from local branch lb
 `abc
 Another file here!
```
Comment (jglin) [inline b.txt:4]: These lines look okay

**Review by jglin — CHANGES_REQUESTED** [review]
Just one small change

**Comment by joeg3** [comment]
I implemented the changes

**Pull request merged**